packages/eslint-config-udemy-website/index.js

```diff
@@ -15,7 +17,8 @@ module.exports = {
     settings: {
         'import/resolver': {
             webpack: {
-                config: 'webpack/config-local.js',
+                // Out of eslint-config-udemy-website, out of node_modules.
```

Comment:
The previous code stopped working because it tries to resolve the webpack directory relative to the library's directory inside node_modules.